--- origsrc/portaudio/Makefile.in	2017-07-20 04:36:34.627993700 -0500
+++ src/portaudio/Makefile.in	2017-07-20 16:16:43.363854500 -0500
@@ -44,7 +44,7 @@ PALIB = libportaudio.la
 PAINC = include/portaudio.h
 
 PA_LDFLAGS = $(LDFLAGS) $(SHARED_FLAGS) -rpath $(libdir) -no-undefined \
-	     -export-symbols-regex "(Pa|PaMacCore|PaJack|PaAlsa|PaAsio|PaOSS)_.*" \
+	     -export-symbols-regex "(Pa|PaMacCore|PaJack|PaAlsa|PaAsio|PaOSS|PaWinDS|PaWinMME)_.*" \
 	     -version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
 
 COMMON_OBJS = \
--- origsrc/portaudio/configure.in	2017-07-20 04:36:34.441475500 -0500
+++ src/portaudio/configure.in	2017-07-20 16:27:08.844127900 -0500
@@ -288,6 +288,7 @@ case "${host_os}" in
         PADLL="portaudio.dll"
         THREAD_CFLAGS="-mthreads"
         CFLAGS="$CFLAGS -I\$(top_srcdir)/src/os/win -DPA_USE_WMME=0 -DPA_USE_ASIO=0 -DPA_USE_WDMKS=0 -DPA_USE_DS=0 -DPA_USE_WASAPI=0"
+        INCLUDES="$INCLUDES pa_win_waveformat.h"
 
         if [[ "x$with_directx" = "xyes" ]]; then
             DXDIR="$with_dxdir"
@@ -297,6 +298,7 @@ case "${host_os}" in
             #VC98="\"/c/Program Files/Microsoft Visual Studio/VC98/Include\""
             #CFLAGS="$CFLAGS -I$VC98 -DPA_NO_WMME -DPA_NO_ASIO"
             CFLAGS="$CFLAGS -I$DXDIR/include -UPA_USE_DS -DPA_USE_DS=1"
+            INCLUDES="$INCLUDES pa_win_ds.h"
         fi
 
         if [[ "x$with_asio" = "xyes" ]]; then
@@ -323,6 +325,7 @@ case "${host_os}" in
             #VC98="\"/c/Program Files/Microsoft Visual Studio/VC98/Include\""
             #CFLAGS="$CFLAGS -I$VC98 -DPA_NO_WMME -DPA_NO_ASIO"
             CFLAGS="$CFLAGS -I$DXDIR/include -UPA_USE_WDMKS -DPA_USE_WDMKS=1"
+            INCLUDES="$INCLUDES pa_win_wdmks.h"
         fi
 
         if [[ "x$with_wmme" = "xyes" ]]; then
@@ -330,6 +333,7 @@ case "${host_os}" in
             LIBS="${LIBS} -lwinmm -lm -lole32 -luuid"
             DLL_LIBS="${DLL_LIBS} -lwinmm"
             CFLAGS="$CFLAGS -UPA_USE_WMME -DPA_USE_WMME=1"
+            INCLUDES="$INCLUDES pa_win_wmme.h"
         fi
 
         if [[ "x$with_wasapi" = "xyes" ]]; then
@@ -337,6 +341,7 @@ case "${host_os}" in
             LIBS="${LIBS} -lwinmm -lm -lole32 -luuid"
             DLL_LIBS="${DLL_LIBS} -lwinmm -lole32"
             CFLAGS="$CFLAGS -I\$(top_srcdir)/src/hostapi/wasapi/mingw-include -UPA_USE_WASAPI -DPA_USE_WASAPI=1"
+            INCLUDES="$INCLUDES pa_win_wasapi.h"
         fi
         ;;
 
